src/tests/Microsoft.PowerFx.Interpreter.Tests/SymbolValueTests.cs

```diff
@@ -752,7 +750,7 @@ public async Task ThisItemTest()
                 var symValuesAll = symTableAll.CreateValues(symValuesThisItem);
 
                 var opts = new ParserOptions { AllowsSideEffects = true };
-                var result = engine.EvalAsync("Set(counter, ThisItem);counter", CancellationToken.None, options: opts, runtimeConfig: symValuesAll).Result;
+                var result = await engine.EvalAsync("Set(counter, ThisItem);counter", CancellationToken.None, options: opts, runtimeConfig: symValuesAll);
```

Comment:
note that await/.Result can change behavior around exception handling. 
But probably fine in this case.